_content/annotated-translation-of-the-chinese-version-of-the-kayagatasati-sutta_kuan-tsefu

```diff
@@ -0,0 +1,19 @@
+---
+title: "Annotated Translation of the Chinese Version of the Kāyagatāsati Sutta"
+authors:
+  - kuan-tsefu
+external_url: "https://www.academia.edu/34449156/Annotated_Translation_of_the_Chinese_Version_of_the_K%C4%81yagat%C4%81sati_Sutta"
+drive_links:
+  - "https://drive.google.com/file/d/1RqMj8T6F0oVb_1I18EjOXtm5q5bf-KLT/view?usp=drivesdk"
+course: sati
+tags:
+  - sutta
```

Comment:
sutta is only for Pāli suttas. Please tag with "ma" instead for the Majjhimā Āgama